site/README.md

```diff
@@ -1,25 +1,25 @@
-# Reamark42 Website
+# Remark42 site
 
-## Work on Your Local Environment
+## Work on your local environment
```

Comment:
here is APA title case, it is used for first/second level of titles. I don't see error here

Reply:
We don't use it in the main Readme file, I think it does make sense to make titles consistent among these two documents.

Reply:
In my opinion it's better be consistent across all of the documentation.

Reply:
@akellbl4 in the main README does this rule also have to be followed?

Reply:
@rakleed nevermind ;)